src/MetricsGenerator.js

```diff
@@ -0,0 +1,18 @@
+const dependenciesMetrics = require('./command/DependenciesMetricsCommand');
+
+const mapMetricToCommand = {
+  dependencies_check: dependenciesMetrics,
+};
+
+const metricsGenerator = argv => {
+  const { metrics } = argv;
+  const metricsList = metrics.split(',');
+
+  metricsList.map(metric => {
+    const command = mapMetricToCommand[metric];
+
+    if (command) command(metric, argv);
```

Comment:
o command é async não? se for, melhor fazer um `filter` e retornando um array de funções e usar o Promise.all